_scss/tutorial/highlight_block.scss

```diff
@@ -22,20 +21,17 @@
 }
 
 @media (min-width: 480px) {
-.steps {
-    .highlight {
-       width: 50%;
-       float: left;
-       max-width: 600px;
-       margin-left: auto;
-       margin-right: auto;
-       margin-bottom: -500px;
-       padding-bottom: 500px;
-    }
-  }
 }
 
 @media (min-width: 768px) {
+  .steps {
+      .highlight {
+         width: 50%;
+         float: left;
+         margin-bottom: -500px;
+         padding-bottom: 500px;
+      }
+    }
 }
```

Comment:
oh ya - I should make a story about testing everything on other browsers ... I've only been using firefox 😬 🙈